Framework/Algorithms/src/SortXAxis2.cpp

```diff
@@ -0,0 +1,152 @@
+#include "MantidAlgorithms/SortXAxis2.h"
+
+using namespace Mantid::Kernel;
+using namespace Mantid::API;
+
+namespace Mantid {
+namespace Algorithms {
+
+DECLARE_ALGORITHM(SortXAxis)
+
+const std::string SortXAxis::name() const { return "SortXAxis"; }
+
+int SortXAxis::version() const { return 2; }
+
+const std::string SortXAxis::category() const {
+  return "Transforms\\Axes;Utility\\Sorting";
+}
+
+const std::string SortXAxis::summary() const {
+  return "Clones the input MatrixWorkspace(s) and orders the x-axis in an "
+         "ascending fashion.";
```

Comment:
Always ascending fashion? It can be descending as well

Reply:
I think this is as well not precise in the documentation of version 1. It's ordering in an ascending (default) or descending fashion.

Reply:
Yep I have corrected it in the cpp version, I'd originally just copied it assuming the python one was correct.

Reply:
Thanks! For the Python guy that was my mistake since I added the descending part :) It should be corrected in the Python version as well if you're kind.